packages/graph/hash_graph_clients/typescript/.openapi-generator/VERSION

```diff
@@ -0,0 +1 @@
+6.0.1
```

Comment:
Newline? Or is this autogenerated?

Reply:
I believe this is autogenerated and based on the generator version. Ory just keeps this as is, [example](https://github.com/ory/sdk/tree/5ece09f0bca734b519df65dfab28909fb1cc21a4/clients/kratos/typescript/.openapi-generator)